example/src/main/tut/typeclass/Show.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: docs
+title:  "Show"
+---
+
+# Show
+
+`Show` presents a safe and explicit alternative to `Object.toString`.
+By providing instances of this typeclass, a type explicitly defines that it can be converted into human-readable output.
+
+**Typical imports**
+
+```tut:silent
+import scalaz.Prelude._
+```
+
+# Built-in instances
+
+```tut
+val stringShow: Show[String] = implicitly
+
+"Scalaz 8".show
+```
+
+# Instance declaration
+
+```tut
+case class Foo(a: Int)
+implicit val fooShow: Show[Foo] = (f: Foo) => s"Foo[${f.a}]"
+```
+
+# Usage
+
+```tut
+val bad = Foo(1).toString
+val good = Foo(1).show
+```
```

Comment:
An example showing that `Show` instances for (say) `Maybe[A]` expects and uses a `Show[A]` rather than just blindly calling `.toString` like `s" ... "` and case class `toString` would be nice.

Even just like in the original `ShowUsage`:

```scala
assert(Maybe.just(Foo(1)).show == "Just(Foo! 1)")
```

Reply:
Oops, I meant to migrate that over!